tools/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -27,14 +26,14 @@ pub async fn subcommand_fix_manifests() -> Result<()> {
 
 struct Manifest {
     path: PathBuf,
-    metadata: cargo_toml::Manifest,
+    metadata: toml::Value,
 }
 
 async fn read_manifests(fs: Fs, manifest_paths: Vec<PathBuf>) -> Result<Vec<Manifest>> {
```

Comment:
to confirm my understanding (and perhaps a comment would be nice), this is copying the version from the actual package onto all the places where the package is used, right? which is why there isn't any hard coded versions or anything in here?

Reply:
That's correct.